server/src/worker/vetted-beneficiary-distribution-worker.ts

```diff
@@ -0,0 +1,14 @@
+import { App } from '../core';
+
+export function runVettedBeneficiaryDistributionWorker(app: App, intervalMilliseconds: number) {
+  async function workLoop() {
+    console.log(`Starting distribution process for vetted beneficiaries at ${new Date()}...`);
+    const result = await app.donationDistributions.distributeDonations({ isVetted: true, beneficiaryStatus: 'verified'});
+    console.log(`Completed distribution process for vetted beneficiaries at ${new Date()}`);
+    console.log(result);
+    console.log();
+    setTimeout(workLoop, intervalMilliseconds);
```

Comment:
You should put this in a try-catch, if it fails due to a lock conflict, you can just log the error, and reschedule it. Or is that already handled else-where?